### ISSUE TITLE
Please update the links to your SDKs and samples

### ISSUE THREAD

**Issue by petersamokhin** [appmetrica/appmetrica-sdk-ios#2]
### 📚 AppMetrica SDK Version

1

### 🛠️ Xcode Version

1

### 📱 Device Information

1

### ⚙️ Installation Method

Other

### 📝 Description

The readme links lead to the legacy pages which lead to the legacy repositories, which are deprecated, archived and not updated since ±2018.

Please update your documentation here: https://appmetrica.yandex.com/docs/ or at least the readme file in this repository.
Adding just a couple of lines with the links to the actual samples will be really useful.

I tried to fill in a report on the documentation website but it requires to provide even more data than this github issue template.

### 🔍 Reproducing the issue

1. Just open literally any of your repositories or documentation pages, on the website or on github
2. See only the deprecations and links to the deprecated pages
3. Feel pain

**Comment by GLinnik21** [comment]
Hello!
I feel your pain, but unfortunately, this issue is not related to the SDK but rather to the web. Therefore, I will close the issue. For any future concerns, please use the [AppMetrica Support Feedback](https://appmetrica.yandex.com/docs/troubleshooting/feedback-new.html). However, I'll pass this issue on to the team to resolve.

**Comment by petersamokhin** [comment]
Readme is a part of the SDK, isn't it? 🙂
Improving it by adding a couple of lines seems to be a legit improvement the engineers could do here in the repo, and I see no other issue template apart from the "bug", thus I hope it will help. Thanks for asking your team anyway

**Comment by GLinnik21** [comment]
Yes, the README seems to have a valid link, but the website has outdated links.  We are working on improving the documentation and creating a more comprehensive README.
